### PR TITLE
tests to delete files

### PR DESCRIPTION
**Pull request by individual-it**
## Description
test if deleting files & folders works correctly

## Related Issue

- part of #844

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by individual-it** [comment]
changed to normal PR as drone seems to have trouble with it otherwise

**Comment by ownclouders** [comment]
Find screenshots at https://minio.owncloud.com/phoenix/screenshots/2184/

**Comment by ownclouders** [comment]
Find screenshots at https://minio.owncloud.com/phoenix/screenshots/2185/

**Comment by ownclouders** [comment]
Find screenshots at https://minio.owncloud.com/phoenix/screenshots/2211/

**Comment by ownclouders** [comment]
Find screenshots at https://minio.owncloud.com/phoenix/screenshots/2217/

**Comment by ownclouders** [comment]
Find screenshots at https://minio.owncloud.com/phoenix/screenshots/2218/

**Comment by individual-it** [comment]
tests currently fail because of broken master-QA build

**Comment by ownclouders** [comment]
Find screenshots at https://minio.owncloud.com/phoenix/screenshots/2225/

**Comment by individual-it** [comment]
hmm looks like the changed in ODS were not pulled in, Delete buttons are not found

**Comment by individual-it** [comment]
tests passing now, ready for review.
after merging #940 the yarn changes are not needed here any-more
the real issue was anyway in https://github.com/owncloud/phoenix/pull/923/files#diff-4d43219c8fdf60539ad2d855346d33f2R36 `:ariaLabel` vs `:arialabel`

File: tests/acceptance/stepDefinitions/filesContext.js [hunk 1]
```diff
@@ -37,27 +38,48 @@ When('the user enables the setting to view hidden folders on the webUI', functio
...
+Then('the deleted elements should not be listed on the webUI', function () {
+  for (const element of deletedElementsTable) {
+    client.page.filesPage().assertElementNotListed(element)
+  }
+  return client
+})
+
+Then('the deleted elements should not be listed on the webUI after a page reload', function () {
+  client.refresh()
+  for (const element of deletedElementsTable) {
```
Comment (phil-davis) [inline tests/acceptance/stepDefinitions/filesContext.js]: Can you call the method above that already does this loop?

Comment (individual-it) [reply tests/acceptance/stepDefinitions/filesContext.js]: that should not be done see https://docs.cucumber.io/guides/anti-patterns/#support-for-conjunction-steps
and I also cannot see how that would work in JS, `Then()` is a function call and not a function
would have to make a outside function that is called from both places

File: tests/acceptance/customCommands/waitForOutstandingAjaxCalls.js [hunk 2]
```diff
@@ -6,7 +6,15 @@ const sleepWhenOutstandingAjaxCalls = function (result) {
 }
 
 exports.command = function () {
+  // init the ajax caounters if they haven't been initialized yet
```
[the remark this answers is not in the document]
Comment (individual-it) [reply tests/acceptance/customCommands/waitForOutstandingAjaxCalls.js]: fixed

File: tests/acceptance/stepDefinitions/filesContext.js (hunk 1 above)
Comment (individual-it) [reply tests/acceptance/stepDefinitions/filesContext.js]: made a small helper function that is called from both steps

**Pull request merged**